tommath.h

```diff
@@ -229,7 +229,7 @@ typedef struct  {
 } mp_int;
 
 /* callback for mp_prime_random, should fill dst with random bytes and return how many read [upto len] */
-typedef int private_mp_prime_callback(unsigned char *dst, int len, void *dat);
+typedef int private_mp_prime_callback(unsigned char *dst, int len, const void *dat);
```

Comment:
This is wrong since it changes the signature of the public (but deprecated) function type ltm_prime_callback.

Reply:
so, this commit must wait for the removal of `ltm_prime_callback` which is currently only deprecated.

Reply:
No. The callback will go away completely in the end.